### PR TITLE
#2105 Parallax sample app does not look good in dark mode

### PR DESCRIPTION
**Pull request by PRINCESANCHEZ**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: canvas_modules/harness/src/client/components/custom-canvases/parallax/parallax.scss [hunk 1]
```diff
@@ -21,6 +21,10 @@
 	// ----------------------------------------------------
 	.d3-node-group {
 
+		.d3-node-label{
+			color: black;
```
Comment (tomlyn) [inline canvas_modules/harness/src/client/components/custom-canvases/parallax/parallax.scss:25]: I thought you were going to us a Carbon token here  - right?

**Review by tomlyn — CHANGES_REQUESTED** [review]

File: canvas_modules/harness/src/client/components/custom-canvases/parallax/parallax.scss (hunk 1 above)
Comment (PRINCESANCHEZ) [reply canvas_modules/harness/src/client/components/custom-canvases/parallax/parallax.scss:25]: I tested all of the `text tokens` but the different shades of gray didn't seem to sit well with some of the lighter nodes.

**Review by tomlyn — CHANGES_REQUESTED** [review]
Using a keyword for the color ('black') is causing a build error. You can use the hex equivalent #000000 instead to allow the build to run.

BTW - You should really be making sure your PR builds OK and tests run successfully before putting it out for review.

**Review by tomlyn — APPROVED** [review]
LGTM

**Pull request merged**